tsconfig.json

```diff
@@ -1,11 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "target": "es6",
```

Comment:
ooh does this decrease our bundle size??

Reply:
Not really sure. I hope so. It was one of the steps I took in debugging errors.